docs/design/disaster-recovery-optimization.md

```diff
@@ -0,0 +1,52 @@
+# FEATURE: Regional Disaster Recovery Optimization
+
+## Summary
+
+Implement use of `bluestore-rdr` as the OSD backend store for customers using ODF with Regional Disaster Recovery (RDR).
+
+## Motivation
+
+- Ceph uses BlueStore as a special-purpose storage back end designed specifically for managing data on disk for Ceph OSD workloads. Ceph team is working on improving this backend storage that will help in significantly improving the performance, especially in disaster recovery scenarios. This new OSD backend is known as bluestore-rdr. In the future, other backends will also need to be supported, such as seastore.
+
+## Proposal
+
+- Enable the optimized OSD backend store for RDR clusters.
+- Enable this optimization for both new (Greenfield) and existing (brownfield) customers.
+
+### Greenfield
+- If `ocs.openshift.io/optimizeDisasterRecovery: true` annotation is available in the `StorageCluster` then configure OSD store type in the `cephCluster` as:
+
+```YAML
+spec:
+  storage:
+    store:
+        type: bluestore-rdr
+```
+
+### Brownfield
+- An existing ODF cluster can be updated to use the new OSD backend store.
+- If `spec.mirroring.enabled` is set to `true`, ODF version is greater than equal to 4.14 and OSD store optimizations are not applied already, then configure OSD store in the `cephCluster` as:
```

Comment:
We can skip version check, since optimization will be applied in whichever version the code is added.
We will never have StorageCluster in 4.13 but operators in 4.14.